DataConnectors/Cybereason/cybereason-pull/__init__.py

```diff
@@ -154,17 +146,20 @@ def post_data(customer_id, shared_key, body, log_type):
 
 
 def main(timer: func.TimerRequest) -> None:
+    utc_timestamp = datetime.utcnow().isoformat()
     if timer.past_due:
         logging.info("The timer is past due!")
+    logging.info('Python timer trigger function ran at %s', utc_timestamp)
     start_time, end_time = generate_date()
     malops = get_detections(start_time, end_time)
     if len(malops["malops"]) > 0:

```

Comment:
Like @arledesma mentioned, this can be reduced to 
```suggestion
    if malops["malops"]:
```

In fact, if you don't need to do all the `logging.info()` calls, you can basically just remove the `if .. else ..` and just loop over the `malops["malops"]` array. Nothing will happen if the array is zero length.